apps/mull-ui/src/app/components/event-card/event-card.scss

```diff
@@ -24,6 +24,7 @@
   border-radius: 0.5rem;
   padding: 0.2rem;
   text-align: center;
+  width: 2.8rem;
```

Comment:
It would be better to add padding instead of hardcoding the width if you want some spacing imo.

Reply:
that has been addressed